packages/core/js-client-isomorphic/src/worker-resolvers/node.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import assert from "assert";
+import module from "module";
+import path from "path";
+import url from "url";
+
+import { Worker } from "threads/master";
+
+export async function getWorker(pkg: string, CDNUrl: string): Promise<Worker> {
+  assert(typeof CDNUrl === "string");
```

Comment:
seems super weired given the ts type is already a string. Change the type in ts or leave the comment and it would be best to have a readable error here in assert's second argument that would explain what might have gone worng and what user should possibly do

Reply:
This is workaround for eslint. This argument is not used and i dont feel good putting comments on such places.
Maybe you have a suggestion how to cope with these cases

Reply:
ok then why to pass it here if it's not used? leave the comment. I think if it's requied for some reason to pass it here and not use - eslint-disable is an exceletn choice instead of doing some weird runtime check that does nothing essentially

Reply:
ah I understand. It's because the other getWorker function needs CNDUrl. Are there problems if this function doesn't have this argument?

Reply:
I think the most clear way of doing things would be to have a separate type definition for both of the functions and use it like this

export const getWorker: GetWorker = () => {}

this way you will not have to include the unused arg and it will still typecheck I think

Reply:
also as an alternative an object literal can be passed so the arguments are named and this way you can choose weather to desctructure it or not inside the function implementation, while the type remains exactly the same

Reply:
These signature should match exactly.

Reply:
I like option with passing object there

Reply:
These signature should match for simplicity and consistency. It would be hard and confusing both to maintain 2 declarations and woth with them from user's pov

Reply:
In order for this signature to match exactly, as I said - the most clear way would be to have a type that you want

`type GetWorker = ...`

that will be used in both places - this way your intent of keeping both signatures the same wouldn't be lost and I believer eslint-disable will not be necessarry in this case

Reply:
Ok, thanks. Will do